iothub/service/src/Feedback/AmqpFeedbackReceiver.cs

```diff
@@ -145,7 +145,7 @@ public override Task CompleteAsync(FeedbackBatch feedback)
                 _faultTolerantReceivingLink,
                 feedback.LockToken,
                 AmqpConstants.AcceptedOutcome,
-                true);
+                false);
```

Comment:
I think it would be useful to add the description that you have in the PR topic as a comment on these methods:
> The service only sends the next c2d feedback message after the previous one has been ack'd, so it doesn't make sense to delay sending the ack in case another feedback message is received.